main.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pandas as pd
+import dxpy as dx
+import datetime
+
+from util import read_yaml, dx_login, send_mail
+
+
+# define number of seqs
+seq = read_yaml('seq.yml')
```

Comment:
are you building the docker image with these included? I think passing them in as environment variables when running the container would be best